lib/YuiRestClient/Widget/DebugLabel.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YuiRestClient::Widget::DebugLabel;
```

Comment:
`debug_label` is a property not a control, we could just use Label and access that field

Reply:
got removed, though, as far as I understand, not really : the label [package](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/08740aeb7c44af90224d5cb5e8d5fb0b702e2820/lib/YuiRestClient/Widget/Label.pm#L12) is looking for a property called "text", and even if we modify it to search for "id" it returns nothing anyway.